packages/sdk/tests/docker-compose.yml

```
@@ -46,6 +46,20 @@ services:
       bellecour-fork-voucher-deployment:
         condition: service_completed_successfully
 
+  bellecour-fork-sharing-contract-upgrade:
+    image: dataprotector-sharing-contract-deployer

```

I think it doesn't change anything on your machine but for Mac users, this is useful:

```suggestion
    image: dataprotector-sharing-contract-deployer
    platform: linux/amd64
```

I'll add that to some other images where it's necessary.
Please check again that it works on your machine :)